.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto
+*.sh text eol=lf
```

Comment:
needed for cloning on Windows and mapping into WSL

Reply:
huh  I'm curious why this isn't needed in other dotnet repos, curious if this is a setting in your git client in windows